lib/plist_parser.dart

```diff
@@ -238,12 +238,12 @@ class PlistParser {
   }
 
   Map _handleDict(XmlElement elem) {
-    var children = elem.children.where(_isElement).cast<XmlElement>();
+    final children = elem.children.where(_isElement).cast<XmlElement>();
 
-    var keys =
-        children.where((el) => el.name.local == 'key').map((el) => el.text);
+    final keys =
+        children.where((el) => el.name.local == 'key').map((el) => el.innerText);
```

Comment:
```suggestion
    final keys = children
        .where((el) => el.name.local == 'key')
        .map((el) => el.innerText);
```

fixed format with dart formatter

Reply:
@dirablue fixed.